themes/powerline/powerline.theme.bash

```diff
@@ -37,13 +37,17 @@ function set_rgb_color {
 }
 
 function powerline_shell_prompt {
+    SHELL_PROMPT_COLOR=${SHELL_THEME_PROMPT_COLOR}
+    if sudo -n true &>/dev/null; then
```

Comment:
This is different from what we're using in the _powerline-multiline_ theme... See https://github.com/Bash-it/bash-it/blob/master/themes%2Fpowerline-multiline%2Fpowerline-multiline.theme.bash#L58

The `sudo -n true` command does not work reliably across operating systems. It doesn't work for me on OS X for example. Abusing the `uptime` command for this purpose (see above link) works across operating systems...